app/waf/src/models/monitor.model.ts

```diff
@@ -0,0 +1,46 @@
+import {Entity, model, property} from '@loopback/repository';
+
+@model()
+export class Monitor extends Entity {
```

Comment:
Suggest to inherit CommonEntity in models/common.model.ts, who helps to define id, name, createdAt and updatedAt properties.